scripts/integer-tests.sh

```diff
@@ -12,6 +12,7 @@ function usage() {
     echo "--signed-only             Run only signed integer tests, by default both signed and unsigned tests are run"
     echo "--cargo-profile           The cargo profile used to build tests"
     echo "--avx512-support          Set to ON to enable avx512"
+    echo "--tfhe-package            The package spec like tfhe@0.4.2, default=tfhe"
```

Comment:
Should we put the last tfhe-rs version in the exemple ? :thinking:

Reply:
does not really matter, it's an example and we won't update it at each release